application/main.py

```diff
@@ -30,30 +30,165 @@
 reload(sys)
 sys.setdefaultencoding("utf-8")
 
+import common
+from config import Config
+from gui.main_window import MainWindow
+
+class Arguments(object):
```

Comment:
An instance of Argument is populated from command lines similar to what was
returned by the old _parse_arguments() in main_window.py - this way I just
had to populate similar fields as Python argparse and could parse it
downstream.


On Tue, Jul 19, 2016 at 1:08 AM, Mladen Mijatov <notifications@github.com>
wrote:

> In application/main.py
> <https://github.com/MeanEYE/Sunflower/pull/235#discussion_r71253534>:
>
> > @@ -30,30 +30,165 @@
> >  reload(sys)
> >  sys.setdefaultencoding("utf-8")
> >
> > +import common
> > +from config import Config
> > +from gui.main_window import MainWindow
> > +
> > +class Arguments(object):
>
> Why have a separate class?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MeanEYE/Sunflower/pull/235/files/5b18d9a8e2a66a68ad574d36f1bd5a4328e144d8#r71253534>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADWvT3gOYeL0uibc0GFOaje2170VOpvuks5qXBVigaJpZM4Iiju1>
> .
>